timemachine/fe/free_energy.py

```diff
@@ -780,7 +780,7 @@ def compute_intermediate_result(lambdas: Sequence[float]) -> PairBarResult:
             if min_overlap is not None:
                 overlap_info = f"Current minimum BAR overlap {cost_to_overlap(max(costs)):.3g} <= {min_overlap:.3g} "
             else:
-                overlap_info = f"Current minimum BAR overlap {cost_to_overlap(max(costs)):.3g} (min_overlap == None)"
+                overlap_info = f"Current minimum BAR overlap {cost_to_overlap(max(costs)):.3g} (min_overlap == None) "
```

Comment:
This looks weird without the space, so added it in even though this is not a common practice for us